### PR TITLE
ref(ui): Replace PlatformIcons component with npm package

### PR DESCRIPTION
**Pull request by matejminar**
We published react platformicons component here: https://github.com/getsentry/platformicons/pull/12.

This PR replaces the custom logic with the new shared one from platformicons npm package.

### CODE REVIEW

**Comment by untitaker** [comment]
#### 📝Changed routes:


| [**/**](https://sentry-docs-git-ref-platformicon.sentry.dev/) | [**/platform-redirect**](https://sentry-docs-git-ref-platformicon.sentry.dev/platform-redirect) |
|:-:|:-:|
|<a href="https://sentry-docs-git-ref-platformicon.sentry.dev/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/ObmX9dBV6itjgFxtJp5TaLn56js_.png">(view full size)</a> |<a href="https://sentry-docs-git-ref-platformicon.sentry.dev/platform-redirect">Screenshot of /platform-redirect</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/w7o7nU9pcXDeW98fToTNUAoXdVk_.png">(view full size)</a> |







Commit 4e31b0d0a1b334c79c985d9f45b13fedc444b14c (https://sentry-docs-git-ref-platformicon.sentry.dev).

File: src/components/platformGrid.tsx [hunk 1]
```diff
@@ -79,7 +79,12 @@ export default ({ noGuides = false }: Props): JSX.Element => {
               <PlatformCell>
                 <PlatformCellIcon>
                   <SmartLink to={platform.url}>
-                    <PlatformIcon size={82} platform={platform.key} />
+                    <PlatformIcon
+                      size={82}
+                      platform={platform.key}
+                      format="lg"
+                      style={{ maxWidth: "none", border: 0, boxShadow: "none" }}
```
Comment (dcramer) [inline src/components/platformGrid.tsx:86]: is this due to default styles in PlatformIcon, or due to stuff its inheriting in docs?

**Review by dcramer — APPROVED** [review]

**Comment by dcramer** [comment]
some margin issues here: https://sentry-docs-git-ref-platformicon.sentry.dev/platforms/

**Comment by matejminar** [comment]
Hrm, I believe it's already in production like that because visually this should be one to one exact copy.

File: src/components/platformGrid.tsx (hunk 1 above)
Comment (matejminar) [reply src/components/platformGrid.tsx:86]: Due to stuff its inheriting. The old local platformIcon component used to apply it everywhere:

https://github.com/getsentry/sentry-https://github.com/getsentry/sentry-docs/blob/860cd098a2b7536946f40ac786b2f0defd10c36a/src/components/platformIcon.tsx#L82-L90

but this is the only place where it makes a real difference.

**Comment by matejminar** [comment]
Yep, it's weird in production right now:

image

I adjusted it and now it should look fine:

image

**Pull request merged**